generic_config_updater/patch_sorter.py

```diff
@@ -544,6 +544,119 @@ def _get_default_value_from_settings(self, parent_path, field_name):
+            ["BGP_MONITORS", "*", "holdtime"],
+            ["BGP_MONITORS", "*", "keepalive"],
+            ["BGP_MONITORS", "*", "name"],
+            ["BGP_MONITORS", "*", "asn"],
+            ["BGP_MONITORS", "*", "local_addr"],
+            ["BGP_MONITORS", "*", "nhopself"],
+            ["BGP_MONITORS", "*", "rrclient"],
+            ["MIRROR_SESSION", "*", "*"],
+        ]
+
+    def get_filter(self):
+        return JsonPointerFilter(self.patterns,
+                                 self.path_addressing)
+
+class RemoveCreateOnlyDependencyFilter(CreateOnlyFilter):
+    def __init__(self, path_addressing):
+        super(RemoveCreateOnlyDependencyFilter, self).__init__(path_addressing)
+        self.patterns = [
+            ["PORT", "*", "lanes"],
+        ]
+
+class RemoveCreateOnlyDependencyMoveValidator:
+    def __init__(self, path_addressing):
+        self.path_addressing = path_addressing
+        self.create_only_filter = RemoveCreateOnlyDependencyFilter(path_addressing).get_filter()
+
+    def validate(self, move, diff):
+        current_config = diff.current_config
+        target_config = diff.target_config # Final config after applying whole patch
+
+        for path in self.create_only_filter.get_paths(current_config):
+            tokens = self.path_addressing.get_path_tokens(path)
+            table_to_check = tokens[0]
+
+            if table_to_check not in current_config:
+                continue
+
+            current_members = current_config[table_to_check]
+            if not current_members:
+                continue
+
+            if table_to_check not in target_config:
+                continue
+
+            target_members = target_config[table_to_check]
+            if not target_members:
+                continue
+
+            simulated_config = move.apply(current_config) # Config after applying just this move
+
+            for member_name in current_members:
+                if member_name not in target_members:
+                    continue
+
+                if not self._validate_member(tokens, member_name,
+                                             current_config, target_config, simulated_config):
+                    return False
+
+        return True
+
+    def _validate_member(self, tokens, member_name, current_config, target_config, simulated_config):
+        table_to_check, create_only_field = tokens[0], tokens[-1]
+
+        current_field = self._get_create_only_field(
+            current_config, table_to_check, member_name, create_only_field)
+        target_field = self._get_create_only_field(
+            target_config, table_to_check, member_name, create_only_field)
+
+        if current_field == target_field:
+            return True
+
+        simulated_member = self.path_addressing.get_from_path(
+            simulated_config, f"/{table_to_check}/{member_name}")
+
+        if simulated_member is None:
+            return True
+
+        if table_to_check == "PORT":
```

Comment:
Hi @qiluo-msft , still need to confirm admin_status need to be set with "down" for PORT["lanes"] change.

Reply:
Yes. PORT["lanes"] are created only field. If the user require to change this field, we need to delete port and create new port. It is already a disruptive change, set `admin_status` to `down` will make it super safe on all platform, and does not make things worse.